dev/app/app.js

```diff
@@ -1,6 +1,11 @@
 'use strict';
 
 /**
+ * Change to switch between local JSON and firebase
+ */
+var localData = true;
+
```

Comment:
They can change it and commit but the `dist` version is always firebase no matter what they change this to.